### PR TITLE
chore(gpu): remove lwe chunk size argument from the multi-bit PBS

### PR DESCRIPTION
**Pull request by agnesLeroy**
closes: _please link all relevant issues_

### Check-list:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Relevant issues are marked as resolved/closed, related issues are linked in the description
* [ ] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

**Comment by agnesLeroy** [comment]
I think we may have had some mismatch with this lwe_chunk_size argument, and maybe it was causing the unspecified launch failure. I'm running the H100 tests several times to check it.

File: backends/tfhe-cuda-backend/cuda/tests_and_benchmarks/benchmarks/benchmark_pbs.cpp [hunk 1]
```diff
@@ -362,9 +358,9 @@ MultiBitPBSBenchmarkGenerateParams(benchmark::internal::Benchmark *b) {
...
   std::vector<MultiBitPBSBenchmarkParams> params = {
       // 4_bits_multi_bit_group_2
-      (MultiBitPBSBenchmarkParams){818, 1, 2048, 22, 1, 1, 2, 0},
+      (MultiBitPBSBenchmarkParams){818, 1, 2048, 22, 1, 1, 2},
       // 4_bits_multi_bit_group_3
-      (MultiBitPBSBenchmarkParams){888, 1, 2048, 21, 1, 1, 3, 0},
+      (MultiBitPBSBenchmarkParams){888, 1, 2048, 21, 1, 1, 3},
   };
 
   // Add to the list of parameters to benchmark
```
Comment (pdroalves) [inline backends/tfhe-cuda-backend/cuda/tests_and_benchmarks/benchmarks/benchmark_pbs.cpp:366]: You need to update this piece of code as well to remove the loop on the lwe_chunk_size.

**Review by pdroalves — APPROVED** [review]
That argument is indeed not needed anymore. 

To remove it you also need to update the function that generates the arguments for the internal benchmark.

**Review by pdroalves — CHANGES_REQUESTED** [review]
Ops, there is actually something to change before approval. Sorry!

**Comment by agnesLeroy** [comment]
I just made the change :slightly_smiling_face:

**Review by pdroalves — APPROVED** [review]

**Pull request merged**